### ISSUE TITLE
Baseband firmware isn't signed for every firmware iPad2,2

### ISSUE THREAD

**Issue by Toygoon** [tihmstar/futurerestore#207]
![2018-02-04 6 43 05](https://user-images.githubusercontent.com/2356036/35776187-5b929de6-09db-11e8-8ca4-b093c9f4cbf4.png)
![2018-02-04 6 45 55](https://user-images.githubusercontent.com/2356036/35776205-abe04d84-09db-11e8-80f2-95dd12bb7523.png)

I can't restore any firmware using futurerestore. Also tried iOS 9.3.5 which is latest version, but it shows error. How can i solve this problem?

**Comment by Lucaso123a** [comment]
It is a bug in tsschecker. Check this out: https://github.com/tihmstar/tsschecker/issues/17
Try to comment out line 136 in tsschecker.c, recompile and restore (not sure if it works).

**Comment by Toygoon** [comment]
Thanks, I'll try it.

**Comment by Danfrid** [comment]
@Toygoon don't forget reply!

**Comment by Toygoon** [comment]
Yup, i'm still working for it. It will take a while due to building error
